domain-model.md

```diff
@@ -0,0 +1,9 @@
+Pseudocode | Objects  | Methods |Messages / Properties | Output
+---- |---- | ---- | ---- | ----
+|| Cohort (class)||this.classes @array||
+|||createCohort(cohortNum)||cohort pushed into this.classes @array
+|||removeCohort()| cohort removed by name but first searched | cohort removed from this.classes @array
+|||addStudent()|add student to cohort|
+|||removeStudent()|remove student from cohort|
+||Students()||studentID@number,firstName&LastName @string|
```

Comment:
Hard to make domain models easy to read & follow.  Maybe use a spreadsheet next time & attach the image.